kapacitor/content.md

```diff
@@ -1,45 +1,112 @@
 # Kapacitor
 
-Kapacitor is a data processing engine. It can process both stream and batch data.
+Kapacitor is an open source data processing engine written in Go. It can process both stream and batch data.
+
+[Kapacitor Official Docs](https://docs.influxdata.com/kapacitor/latest/introduction/getting_started/)
 
 %%LOGO%%
 
 ## Using this image
 
+
+### Using the default configuration
+
 Start the Kapacitor container with default options:
 
 ```console
-$ docker run --net=host \
-      -v /path/on/host/kapacitorFiles/:/var/lib/kapacitor
-      kapacitor
+$ docker run -p 9092:9092 \
```

Comment:
They converted all of the multiline examples into a single line so we should probably do the same here.